spec/time_for_a_boolean_spec.rb

```diff
@@ -81,7 +82,7 @@
 
       it 'is false if the attribute date is in the future' do
         klass.time_for_a_boolean :attribute, :attribute_on
-        allow(object).to receive(:attribute_on).and_return(Date.current + 1)
+        allow(object).to receive(:attribute_on).and_return(Time.current + 1)
```

Comment:
Yes.  It was "impossible" to get a bogus test failure when we gave ourselves a day's worth of leeway.  Now that we're down to one second of leeway, it's only "slightly impossible."

In practice, it becomes an issue when someone gives that `allow` wider scope (say, by moving it the beginning of the `describe` block), and then the entire block takes more than a second to run. Better to force a reevaluation every time `attribute_on` is `receive`d.